### PR TITLE
Add basic support for CtrlP

### PR DESCRIPTION
**Pull request by bdielman**
I figure this would be a nice addition. It will highlight the matched words when typing a search using CtrlP.

### Before:
Screen Shot 2019-03-20 at 12 37 38 PM

### After:
Screen Shot 2019-03-20 at 12 31 33 PM

### CODE REVIEW

File: after/plugin/dracula.vim [hunk 1]
```diff
@@ -31,5 +31,10 @@ if exists('g:ale_enabled')
   hi! link ALEInfoSign DraculaCyan
 endif
 " }}}
+"}}}
+" CtrlP: {{{
+  hi! link CtrlPMatch Keyword
```
Comment (dsifford) [inline after/plugin/dracula.vim]: 2 quick things:

1. Can we add an `exists` check here to test for the existence of ctrlp before creating those highlight groups.
2. Can we switch from `Keyword` to `IncSearch` so there's better uniformity with other means of searching (FZF, builtin search, etc)

Comment (bdielman) [reply after/plugin/dracula.vim]: Sounds good! I'll make those changes.

Comment (bdielman) [reply after/plugin/dracula.vim]: Just pushed the requested changes.

**Comment by dsifford** [comment]
Great work. Thanks!

**Comment by bdielman** [comment]
Thanks! 🎉

**Pull request merged**